ansible/roles/jenkins-worker/tasks/main.yml

```diff
@@ -19,6 +19,9 @@
     state: present
     line: ::1 localhost.localdomain localhost
 
+- name: create NODE_TEST_DIR directory
+  file: path=/home/{{ server_user }}/tmp state=directory
```

Comment:
Yeah, you're right @gibfahn. I fixed it adding an extra setup. Now permission dir look like this:
```
drw-r-xr-x  2 iojs iojs 4.0K May  5 17:48 tmp
```